### PR TITLE
updated bds231992 meme caption file

### PR DESCRIPTION
**Pull request by bds231992**
Updated caption text meme assigned to bds231992
updated file: https://github.com/githubschool/spooky-baseball/blob/master/_posts/0000-01-07-bds231992.md

### CODE REVIEW

**Comment by pbreslin** [comment]
Great Job!

**Review by pbreslin — APPROVED** [review]
Good Job

File: _posts/0000-01-07-bds231992.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![bad-luck-brian](https://cloud.githubusercontent.com/assets/16547949/25400743/8ae447a2-29c1-11e7-9839-5438d8fcda7b.jpg)
 {: .center}
 
-Caption here
+Merges one PR.. breaks everything ever..
```
Comment (bds231992) [inline _posts/0000-01-07-bds231992.md]: good&funny (y)  yes

File: _posts/0000-01-07-bds231992.md [hunk 2]
```diff
@@ -8,6 +8,3 @@ title: "Welcome to our slide deck!"
 
 Merges one PR.. breaks everything ever...
```
Comment (bds231992) [inline _posts/0000-01-07-bds231992.md]: Bad practice @bds231992.. #shame

**Pull request merged**